### ISSUE TITLE
[SF_ACC][1.7.1][Table] JAWS does not announce the selected/unselected status of row

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#5840]
### **Bug Description**
JAWS does not announce the selected/unselected status of row.

### **Expected Behavior**
Screen reader should announce the selected or unselected status of table row.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Table/
2. Select some rows
3. Use arrow keys to go through table rows
4. JAWS does not announce the selected or unselected status of row

### **Isolated Example**
https://sap.github.io/ui5-webcomponents/playground/components/Table/

### **Context**
- UI5 Web Components version: 1.7.1
- OS/Platform: Windows10
- Browser: Chrome
- Affected component: Table
- Screen Reader: JAWS2022

### **Log Output / Stack Trace / Screenshots**
![Partially_Fixed](https://user-images.githubusercontent.com/30744017/191709886-576f83ce-ff25-4808-a88e-fa48700b9dfc.png)

### **Priority**
- [ ] Low
- [ ] Medium
- [x] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-rl , 

It looks like the JAWS Reader doesn't announce whether or not the table row is selected even though there is such html attribute on the DOM node of the row.

I also noticed that the List component doesn't have that problem.

Could you please look into it ?

Regards,
Plamen Ivanov
